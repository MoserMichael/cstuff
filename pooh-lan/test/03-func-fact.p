
# recursive function call

sub fact( n )
  if n > 1
    return n * fact( ~n n - 1 )
  else 
    return 1
  end
end

for i [ 1, 5, 15 ]
   println ( ~msg 'Factorial of [ i ]  = [ fact( ~n i ) ] ' )
end

for i [ 1, 5, 15 ]
  println ( ~msg 'Factorial of ' ..  i .. '  =  ' .. fact( ~n i )  )
end
