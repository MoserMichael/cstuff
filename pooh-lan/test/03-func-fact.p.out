sh: ../GNU-Linux-3.0.0-12-generic-i686/bin/parser: not found
